calm-integrations/ey-calm-migration-scripts/ey-post-migration-script.py

```diff
@@ -0,0 +1,144 @@
+        log.info('Response: {}'.format(json.dumps(json.loads(resp.content), indent=4)))
+        raise
+
+def main():
+    try:
+        total_vms = 1
+        offset = 0
+
+        init_contexts()
+        create_session()
+
+        while offset < total_vms:
+            vm_entities, total_vms = get_vm_list(dest_base_url, dest_pc_auth, offset)
+        
+            for vm in vm_entities:
+                instance_id = vm["metadata"]["uuid"]
+                NSE = model.NutanixSubstrateElement.query(instance_id=instance_id)
+                if NSE:
+                    NSE = NSE[0]
+                    if NSE.spec.resources.account_uuid != dest_account_uuid:
+                        print instance_id
+                        vm_spec = get_vm(dest_base_url, dest_pc_auth, instance_id)
+                        NSE.spec.resources.account_uuid = dest_account_uuid
+                        NSE.spec.resources.cluster_uuid = vm_spec["status"]["cluster_reference"]["uuid"]
+                        for i in range(len(NSE.spec.resources.nic_list)):
+                            NSE.spec.resources.nic_list[i].nic_type = vm_spec["status"]["resources"]["nic_list"][i]["nic_type"]
+                            NSE.spec.resources.nic_list[i].subnet_reference = vm_spec["status"]["resources"]["nic_list"][i]["subnet_reference"]
+                            NSE.spec.resources.nic_list[i].ip_endpoint_list = vm_spec["status"]["resources"]["nic_list"][i]["ip_endpoint_list"]
+                        NSE.save()
+                        NS = NSE.replica_group
+                        NS.spec.resources.account_uuid = dest_account_uuid
+                        for i in range(len(NS.spec.resources.nic_list)):
+                            NS.spec.resources.nic_list[i].nic_type = vm_spec["status"]["resources"]["nic_list"][i]["nic_type"]
+                            NS.spec.resources.nic_list[i].subnet_reference = vm_spec["status"]["resources"]["nic_list"][i]["subnet_reference"]
+                            NS.spec.resources.nic_list[i].ip_endpoint_list = vm_spec["status"]["resources"]["nic_list"][i]["ip_endpoint_list"]
+                        NS.save()
+                        NSC = NS.config
+                        NSC.spec.resources.account_uuid = dest_account_uuid
+                        for i in range(len(NSC.spec.resources.nic_list)):
+                            NSC.spec.resources.nic_list[i].nic_type = vm_spec["status"]["resources"]["nic_list"][i]["nic_type"]
+                            NSC.spec.resources.nic_list[i].subnet_reference = vm_spec["status"]["resources"]["nic_list"][i]["subnet_reference"]
+                            NSC.spec.resources.nic_list[i].ip_endpoint_list = vm_spec["status"]["resources"]["nic_list"][i]["ip_endpoint_list"]
+                        NSC.save()
+                        flush_session()
+                        app_name = model.Application.get_object(NSE.application_reference).name
```

Comment:
`NSE.application.name` will give you app name. 
Also we should create set of names and do it in separate loop.